packages/sync-actions/test/types-sync-enums.spec.js

```diff
@@ -301,11 +297,7 @@ describe('Actions', () => {
           {
             action: 'changeLocalizedEnumValueOrder',
             fieldName: 'lenum',
-            keys: [
-              { key: 'enum_4', label: { en: 'enum-4' } },
-              { key: 'enum_1', label: { en: 'enum-1' } },
-              { key: 'enum_2', label: { en: 'enum-2' } },
-            ],
+            keys: ['enum_4', 'enum_1', 'enum_2'],
```

Comment:
Here is docs
https://docs.commercetools.com/api/projects/types#add-localizedenumvalue-to-fielddefinition